tests/pipeline/test_config_isolation.py

```
@@ -0,0 +1,180 @@
+    `GIT_CONFIG_GLOBAL` points at a configuration file, which is how tooling
+    (this harness included) deliberately controls Git. If sanitization removed
+    it, every sealed test would silently fall back to the real `~/.gitconfig`.
+    """
+    world = repo_scenario()
+    marker = world.parent_path.parent / "marker.gitconfig"
+    marker.write_text("[user]\n\tname = sentinel-from-file-pointer\n")
+    env = dict(world.env)
+    env["GIT_CONFIG_GLOBAL"] = str(marker)
+
+    observed = (
+        subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)